### ISSUE TITLE
Remove unused CSS classes (button colors, .arrow-right, .arrow-left)

### ISSUE THREAD

**Issue by BraadMartin** [MIGHTYminnow/trestle#25]
Should be more like:

```
.arrow-right::before,
.arrow-right::after {
	display: inline-block;
	line-height: 1;
	font-family: FontAwesome;
	font-weight: normal;
	font-stretch: normal;
	font-style: normal;
	font-variant: normal;
	-webkit-font-smoothing: antialiased;
}

.arrow-right::after {
	content: "\f0a9";
}

.arrow-left::before {
	content: "\f0a8";
}
```

**Comment by MickeyKay** [comment]
I made this edit in the most recent commit which we reverted, however I now think we should remove this entirely. The Trestle button shortcode allows for easy insertion of icons in the markup, and more often we end up styling buttons on a per-site basis. I've not incorporated the above styles on a single site we've built, and I recommend we remove them entirely.

Yay? Nay? If you concur, I can go ahead and remove them entirely.
